R/calcIEA_ETP.R

```diff
@@ -44,6 +46,18 @@ calcIEA_ETP <- function() {
     return(x)
   }
 
+  # keep only countries that were reported individually or are part of given regions
+  .removeRegions <- function(x, keepRegions) {
+    map <- toolGetMapping("regionmappingIEA_ETP.csv",
+                          where = "mrremind", type = "regional",
+                          returnPathOnly = TRUE) %>%
+      read.csv2(check.names = FALSE)
+    keepCountries <- map %>%
+      filter(if_any(c("individual", keepRegions))) %>%
```

Comment:
This line throws a warning for me

```
Warning message:
Using an external vector in selections was deprecated in tidyselect 1.1.0.
ℹ Please use `all_of()` or `any_of()` instead.
  # Was:
  data %>% select(keepRegions)

  # Now:
  data %>% select(all_of(keepRegions))
  
  ```

Reply:
Thanks for spotting, I missed this but fixed it now.